internal/controllers/machine/drain/drain.go

```diff
@@ -448,46 +448,38 @@ func (r EvictionResult) ConditionMessage(nodeDrainStartTime *metav1.Time) string
 
 	conditionMessage := fmt.Sprintf("Drain not completed yet (started at %s):", nodeDrainStartTime.Format(time.RFC3339))
 	if len(r.PodsDeletionTimestampSet) > 0 {
-		conditionMessage = fmt.Sprintf("%s\n* Pods with deletionTimestamp that still exist: %s",
+		conditionMessage = fmt.Sprintf("%s\n* Pod %s with deletionTimestamp and still not removed from the node",
```

Comment:
```suggestion
		conditionMessage = fmt.Sprintf("%s\n* Pod %s with deletionTimestamp and still not removed from the Node",
```

Reply:
Also would recommend to rephrase so we end up with something like this (for consistency with eviction errors)

```yaml
* Pods pod-2-deletionTimestamp-set-1, pod-3-to-trigger-eviction-successfully-1: deletionTimestamp set, but still not removed from the Node
```